SparkPost/2web2ui
# FE-1164 - Update SupportForm to stack content in the success state

[FE-1164](https://sparkpost.atlassian.net/browse/FE-1164)

### What Changed
- Incorporates `<Stack/>` component to address UI bug

### How To Test
- Open the support modal and submit a ticket - verify the success state renders with content stacked.

### To Do
